jaggeryapps/dashboard/serviceproviders/addsp.jag

```diff
@@ -137,16 +137,6 @@ include("../templates/leftpane.jag");
                     </div>
                 </div>
                 <div class="col-xs-12 col-sm-6 col-md-3 col-lg-2">
-                    <div class="cloud-app-type  app-color-two" id="7.0"   data-description="A service provider is a website that hosts applications. You can enable WSO2 Cloud as a service provider. Your users can then access other applications directly from WSO2 Cloud using single sign-on. Single sign-on can be a great help to your users: instead of having to remember many passwords, they only have to remember one. Plus, the applications can be added as tabs to your WSO2 organization, which means users don’t have to switch between programs." data-appname="WSO2 Cloud" >
-                        <div class="thumbnail icon">
-                            <div class="square-element">
-                                <i class="fw fw-wso2 fw-custom-x"></i>
-                            </div>
-                        </div>
-                        <div class="type-name">WSO2 Cloud</div>
-                    </div>
-                </div>
-                <div class="col-xs-12 col-sm-6 col-md-3 col-lg-2">
```

Comment:
This shows , it delete the first line of bellow div block. But its not actually delete bellow line